### ISSUE TITLE
set_external_file_reputation

### ISSUE THREAD

**Issue by binhxe** [opendxl/opendxl-tie-client-python#20]
Could not set comment with set_external_file_reputation.
To reproduce:  run the basic_set_external_file_reputation.py

**Comment by hbazan** [comment]
Hi binhxe, but you do see the reputation changed and missing the comment?

**Comment by binhxe** [comment]
Yes, the reputation is changed and the comment is empty. Additionally, if the filename (optional parameter) is empty, the reputation is not set

**Comment by hbazan** [comment]
You are right, the comment is missing. I will file a bug on TIE Server to track this.
